ddtrace/tracer.py

```diff
@@ -490,7 +490,7 @@ def _start_span(
             if isinstance(child_of, Context):
                 context = child_of
             else:
-                context = child_of.context
+                context = child_of._context or Context()
```

Comment:
if we have a Span without a valid `_context`, should we still have:

`Context(trace_id=child_of.trace_id, span_id=child_of.span_id)`, etc ?

Reply:
oh, why do this instead of `child_of.context`?

Reply:
Yup we can just leave it as `child_of.context` with the changes now